atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/NameMetadataDescription.java

```diff
@@ -181,55 +174,6 @@ public String renderToJson(byte[] name) {
         return sb.toString();
     }
 
-    public byte[] parseFromJson(String json, boolean allowPrefix) {
```

Comment:
This is used by large internal product's command line interface to an AtlasDB database, so I'm not sure we can remove this?

Reply:
This deletion is not needed after #4821